gui/src/components/Backups.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+    <div>
+        <h4 class="pt-3">Backups in Cluster: {{cluster.cluster_name}}</h4>
+        <ClusterSummary :cluster="cluster"/>
+        <div class="row justify-content-end mb-3 pr-3">
```

Comment:
I think we need a bit more space in between the table and the buttons.
(the table for cluster information and the buttons for `Create Backup` and `View Restore`.)
Ditto for the restore page as well.

Reply:
Is it okay for the `Clusters` page?